accepted/00000-hub-reporting.md

```diff
@@ -70,20 +56,50 @@ from the main Uyuni Server database.
 Data which belong to the traditional stack only (like osad status) should not be made available in the
 reporting database as the traditional stack will be deprecated soon.
 
+## Installation
+The reporting tools will be installed by default; all the upgrade to a newer version, will install the 
+reporting tools as well.
 
-## Adding Data to the Reporting DB
+We provide a new package installed in the hub system:
+- to setup the reporting DB schema,
+- to setup and manage the reporting database,
+- to setup and manage DBs in all Uyuni Server.
+
+The management tools should support:
+- Database setup and schema initialization
+- Database schema migration
+- Simple User management. The default setup will create two users:
```

Comment:
Do we want this management feature to allow creating both read-only and read-write users or only the former?

Reply:
I haven't a strong opinion on that, but if I need to choose I would say both read-only and read-write, in order to provide all the users required by the reporting tool

Reply:
I think we should automatically create only the read/write user as we need it directly.
The hub should create his own user when connecting to the Server DB. We described it below.
We should document how to create a user for a reporting tool and the admin decide if he do this on the single server or the hub or both/all.
What do you think?

Reply:
Yes indeed makes sense to leave the final user full control on their accounts, so +1 also for me.

Reply:
Changed this section and add more information below regarding how to use data collected in a reporting tool and documentation